### ISSUE TITLE
Suggestion: add mention to \pgfinterruptpath in \pgfextra's documentation

### ISSUE THREAD

**Issue by user202729** [pgf-tikz/pgf#1327]
As in the title. If I understood correctly that's how it should be used.

https://tikz.dev/base-scopes#pgf.pgfinterruptpath

Side note: the answer in [Adding Tikz node in pgfextra triggers double drawing of containing path - TeX - LaTeX Stack Exchange](https://tex.stackexchange.com/questions/131077/adding-tikz-node-in-pgfextra-triggers-double-drawing-of-containing-path) uses `\pgfinterruptpath` without the environment form. I'm not sure if that is supposed to be supported.

**Comment by muzimuzhi** [comment]
Yes the portable form `\pgfinterruptpath ... \endpgfinterruptpath` is supported and used by tikz itself, for example in `\tikz@finish`:
https://github.com/pgf-tikz/pgf/blob/12f17834c849a659b8829bbc9122bc459f585917/tex/generic/pgf/frontendlayer/tikz/tikz.code.tex#L2365-L2367
